diff --git a/util/sokol_imgui.h b/util/sokol_imgui.h
--- a/util/sokol_imgui.h
+++ b/util/sokol_imgui.h
@@ -2326,7 +2326,7 @@ static void _simgui_create_fonts_texture(void) {
         _simgui_clear(&img_desc, sizeof(img_desc));
         img_desc.image = _simgui.font_img;
         img_desc.sampler = _simgui.font_smp;
         _simgui.default_font = simgui_make_image(&img_desc);
-        io->Fonts->TexID = simgui_imtextureid(_simgui.default_font);
+        io->Fonts->TexID = (ImTextureID)(uintptr_t)simgui_imtextureid(_simgui.default_font);
     }
 
     // a default user image

